garde/tests/rules/length.rs

```diff
@@ -42,9 +42,9 @@ fn length_invalid() {
 
```

Comment:
`range` now also supports `equal`, so please add the tests you added for `length` to `range` as well.

Reply:
Ok will do